server/backend/api/app/routers/router_guards.py

```diff
@@ -153,26 +163,33 @@ async def get_request_role_from_id(
     Some endpoints has path_params with "user_role_xref_id".
     Some endpoints has role_id in request body.
     """
-    user_role_xref_id = None
+    role = None
+
     if "user_role_xref_id" in request.path_params:
         user_role_xref_id = request.path_params["user_role_xref_id"]
-
-    if user_role_xref_id:
         LOGGER.debug(f"Retrieving role by user_role_xref_id: " f"{user_role_xref_id}")
         user_role = crud_user_role.find_by_id(db, user_role_xref_id)
-        return user_role.role
-
+        role = user_role.role
     else:
         try:
             rbody = await request.json()
             role_id = rbody["role_id"]
             LOGGER.debug(f"Retrieving role by Request's role_id: {role_id}")
-            role = crud_role.get_role(db, role_id)
-            return role  # role could be None.
+            role = crud_role.get_role(db, role_id)  # role could be None.
+        except json.JSONDecodeError:  # When request does not contains body part.
```

Comment:
I can't recall since when we started using `try - except json.JSONDecodeError`...
I think this `try-except` is not needed.
The `Depends` method "assume" to get the id from two sources: The "**path_params**" and the json "**body**", then they should be there for the endpoint guard. If the body or path_params are malformed then this is not expected and just let the function raise/throw the original exception.